map_gui/src/render/road.rs

```diff
@@ -91,24 +91,22 @@ impl DrawRoad {
                     app.cs()
                         .zoomed_road_surface(LaneType::Driving, r.get_rank())
                 };
-
-                if false {
-                    // TODO Not ready yet
+                // TODO: find a good way to draw an appropriate background
+                if r.center_pts.quadrant() > 1 && r.center_pts.quadrant() < 4 {
+                    println!("quadrant: {}", r.center_pts.quadrant().to_string());
```

Comment:
Remember to remove these before merging. And slight simplification, you can take out the `to_string()` here -- when you print something with `{}`, it calls https://doc.rust-lang.org/std/fmt/trait.Display.html by default